java-compiler-testing/src/test/java/io/github/ascopes/jct/tests/unit/filemanagers/package-info.java

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright (C) 2022 - 2023 Badrri Narayanan S
```

Comment:
So right now, the license header plugin currently enforces my name is at the top of each file, which means this will fail the CI workflow.

I've put a PR up at https://github.com/ascopes/java-compiler-testing/pull/233 to make this so that it retains the ownership to the original authors. If you rebase onto that when I merge this, it will remove the need for specific names in files (since the commit history should show the original authors directly).

Hope that is all okay!

Reply:
No problem. I will change it to your name. The work is its own reward plus your mentorship :)